ExampleCodes/FFTW/Basic/main.cpp

```diff
@@ -116,8 +120,8 @@ int main (int argc, char* argv[])
             Real x = (i+0.5) * dx[0];
             Real y = (j+0.5) * dx[1];
             Real z = (k+0.5) * dx[2];
```

Comment:
```suggestion
            Real z = (AMREX_SPACEDIM==3) ? (k+0.5) * dx[2] : 0.;
```

Reply:
We need to make sure the code works properly for DIM=2 and DIM=3 builds.
For the DIM=2 case, we basically need to set z equal to zero and make sure the function for phi works properly.